### DISCUSSION TITLE
postal 2.1.1 fast_server install

### DISCUSSION THREAD

**Discussion by ofcakir** [postalserver/postal#2021] [Help with using Postal]
hi everyone 

I could not set up a fast server for postal. What should be added to postal.yml for version 2.1.1.
I want to use it for track

**Comment by willpower232** [comment]
The fast server was removed and click tracking integrated into the main process. Have a look at https://docs.postalserver.io/install/upgrade-to-v2#configuring-web-proxy-for-openclick-tracking
